recipes/sokol/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   "cci.20210304":
     url: "https://github.com/floooh/sokol/archive/c5d55faa1036af9919bd28400e97d37548bb3789.zip"
     sha256: e61c782e287543ffb8077ac24a7bfee05e867ee27f6c3e9487a2dbfa1ca2b466
+  "cci.20230925":
+    url: "https://github.com/floooh/sokol/archive/5633a0f91bf3354320837ce0c7b4748a953a1401.zip"
+    sha256: 9c854bc7e6f73024a99554e8e1c8850fddfc325bb2b1ece2c59502c0e3cef68d
```

Comment:
Newer versions should be put first, at the top of the list. If you could make that change here and in the `config.yml` that would be great.

Reply:
Thanks!

Reply:
I've just pushed a fix in https://github.com/conan-io/conan-center-index/pull/20250/commits/59cb4934404ea0d9d5c3f54a5f398c54a2b336ec, thanks